pkg/kubectl/cmd/drain.go

```diff
@@ -644,7 +643,7 @@ func (o *DrainOptions) waitForDelete(pods []api.Pod, interval, timeout time.Dura
 
 // SupportEviction uses Discovery API to find out if the server support eviction subresource
 // If support, it will return its groupVersion; Otherwise, it will return ""
-func SupportEviction(clientset internalclientset.Interface) (string, error) {
+func SupportEviction(clientset *kubernetes.Clientset) (string, error) {
```

Comment:
We also have kubernetes.Interface. For testability we usually use that instead of the struct pointer.

Reply:
ok, will update it